src/safariservices.cs

```diff
@@ -51,20 +79,29 @@ interface SFSafariViewController {
 		[PostGet ("NibBundle")]
 		IntPtr Constructor ([NullAllowed] string nibName, [NullAllowed] NSBundle bundle);
 
-		[DesignatedInitializer]
+		[Deprecated (PlatformName.iOS, 10, 0, message: "Please use initWithURL:configuration:")]
 		[Export ("initWithURL:entersReaderIfAvailable:")]
```

Comment:
This message would not make sense to a C# developer, replace with the proper constructor signature.